gr-digital/python/digital/qa_packet_format.py

```diff
@@ -63,7 +62,7 @@ def test_packet_format_async_default(self):
         rx_access_code = header[0:len(access_code)]
 
         length = len(send_str)
-        rx_length = struct.unpack_from(b"!H", six.b(header), len(access_code))[0]
+        rx_length = struct.unpack_from(b"!H", header.encode("latin-1"), len(access_code))[0]
```

Comment:
I don't think latin-1 is a good choice. Given how this string is constructed, use `ascii` instead.

Reply:
According to six documentation, should be latin=1, should I change it to ascii instead?
https://six.readthedocs.io/#six.b

Reply:
Ok, now I get your point about "ascii". Header is constructed from pmt.u8vector, so it should have values of 0-255, then encode("ascii") would be additional "assert" for proper u8vector values.

Reply:
Wel ... 

```
rx_length = struct.unpack_from(b"!H", header.encode("ascii"), len(access_code))[0]
UnicodeEncodeError: 'ascii' codec can't encode characters in position 0-5: ordinal not in range(128)
```

From python docs:

> If the code point is 128 or greater, the Unicode string can’t be represented in this encoding. (Python raises a UnicodeEncodeError exception in this case.)

Changing back to latin-1:

> Latin-1, also known as ISO-8859-1, is a similar encoding. Unicode code points 0–255 are identical to the Latin-1 values, so converting to this encoding simply requires converting code points to byte values; if a code point larger than 255 is encountered, the string can’t be encoded into Latin-1.

Reply:
Fair play. I'm still not happy about this encoding/decoding, but that's beside the point of this PR, which is about removing six. We should probably directly use `struct`.